readme.md

```diff
@@ -2,41 +2,56 @@
 
 [![CircleCI](https://circleci.com/gh/ap4k/ap4k.svg?style=svg)](https://circleci.com/gh/ap4k/ap4k)
 
-Annotation processor for Kubernetes is a collection of Java annotation processors and tools for generating Kubernetes/Openshift resources at compile time.
+Ap4k is a collection of Java annotations and processors for generating Kubernetes/Openshift manifests at compile time.
```

Comment:
Why do you use `manifest` as word instead of `resource` which is the exact technical term ?

Reply:
imho manifest strongly suggests that we are referring to the text representation if the resource. Resource feels a little bit more abstract.

Reply:
If you want to keep this word, then I propose to add a few definitions top of the document to explain what is a manifest, resources, ... Make sense @iocanel @metacosm @geoand

Reply:
To tell you the truth, I consider the two words interchangeable, but indeed semantically `manifest` is more appropriate

Reply:
@cmoulliard: I prefer using the term that needs less explaining. If this term is resources, then I'll start using that.

wdyt?

Reply:
You can use it but then explain what is a manifest and what is generated